tests/Benchmark/ConstructBench.php

```diff
@@ -0,0 +1,176 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @copyright     Copyright (c) Brian Nesbitt <brian@nesbot.com>
```

Comment:
Since this is a new file it doesn't need Brian's copyright.

Reply:
Do you have an official header to copy somewhere?

Reply:
You only need the copyright line from line 10.